src/modules/Common/managers/UserManager.tsx

```diff
@@ -107,3 +107,13 @@ export const batchUpsertFromScraping = async (users: any[], platformId: string)
     throw e;
   }
 };
+
+export const list = async () => {
+  try {
+    const users: User[] = await UserModel.find().sort({ username: 1 }).limit(10000);
```

Comment:
it's ok for now but this should definitely be in the calling function

as one may want to use the full list in another part of the code